File: deviceinfo/src/main/java/com/an/deviceinfo/userapps/UserApps.java
package com.an.deviceinfo.userapps;


import org.json.JSONObject;

import java.io.Serializable;


public class UserApps implements Serializable {

    private String appName;
    private String packageName;
    private String versionName;
    private int versionCode;

    public String getAppName() {
        return appName;
    }

    public void setAppName(String appName) {
        this.appName = appName;
    }

    public String getPackageName() {
        return packageName;
    }

    public void setPackageName(String packageName) {
        this.packageName = packageName;
    }

    public String getVersionName() {
        return versionName;
    }

    public void setVersionName(String versionName) {
        this.versionName = versionName;
    }

    public int getVersionCode() {
        return versionCode;
    }

    public void setVersionCode(int versionCode) {
        this.versionCode = versionCode;
    }

    public JSONObject toJSON() {
        JSONObject jsonObject = new JSONObject();
        try {
            jsonObject.put("appName", getAppName());
            jsonObject.put("packageName", getPackageName());
            jsonObject.put("versionName", getVersionName());
            jsonObject.put("versionCode", getVersionCode());
            return jsonObject;

        } catch (Exception e) {
            e.printStackTrace();
            return null;
        }
    }
}

File: deviceinfo/src/main/java/com/an/deviceinfo/userapps/UserApps.kt
package com.an.deviceinfo.userapps


import org.json.JSONObject

import java.io.Serializable


class UserApps : Serializable {

    var appName: String? = null
    var packageName: String? = null
    var versionName: String? = null
    var versionCode: Int = 0

    fun toJSON(): JSONObject? {
        val jsonObject = JSONObject()
        try {
            jsonObject.put("appName", appName)
            jsonObject.put("packageName", packageName)
            jsonObject.put("versionName", versionName)
            jsonObject.put("versionCode", versionCode)
            return jsonObject

        } catch (e: Exception) {
            e.printStackTrace()
            return null
        }

    }
}
